backend/src/api/routes/company-route.ts

```diff
@@ -30,4 +57,87 @@ router.patch(
   }),
 );
 
+router.get(
+  '/okr',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  run((req): Promise<OKR[]> => getAllOkr(req.userId)),
+);
+
+router.get(
+  '/okr/:id',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  run((req): Promise<OKR> => getOkrById(req.params.id)),
+);
+
+router.post(
+  '/okr',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  validateBody(createOkrSchema),
+  run((req): Promise<OKR> => {
+    const { userId, body } = req;
+    const data = { userId, body };
+    return createOkr(data);
+  }),
+);
+
+router.put(
+  '/okr/:id',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  validateBody(updateOkrSchema),
+  run((req): Promise<OKR> => {
+    const { id } = req.params;
+    const { body } = req;
+
+    const data = {
+      okrId: id,
+      data: body,
+    };
+    return updateOkrById(data);
+  }),
+);
+
+router.post(
+  '/okr/:id/objective',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  validateBody(createObjectiveSchema),
+  run((req): Promise<OKR> => {
+    const { body } = req;
+    const { id } = req.params;
+    const data = { okrId: id, body };
+    return createObjectiveToOkr(data);
+  }),
+);
+
+router.put(
+  '/okr/:id/objective/:id1',
+  validatePermissions([RoleType.Admin, RoleType.User]),
+  validateBody(updateObjectiveSchema),
+  run((req): Promise<OKR> => {
+    const { id, id1 } = req.params;
```

Comment:
use meaningful param names
```suggestion
  '/okr/:okrId/objective/:objectiveId',
  validatePermissions([RoleType.Admin, RoleType.User]),
  validateBody(updateObjectiveSchema),
  run((req): Promise<OKR> => {
    const { okrId, objectiveId } = req.params;
```